tests/output/self_hosted/exit_error.txt

```diff
@@ -0,0 +1,2 @@
+Should get printed
+Program 'tests/.oomph-cache/exit_error_compilation0/exit_error' exited with status 1
```

Comment:
This isn't great, because when running tests in parallel, `blah_compilation0` might be instead `blah_compilation1` or something